### ISSUE TITLE
Error Deploying CustomMetadataLoader Into Production

### ISSUE THREAD

**Issue by Aymen-Mtir** [forcedotcom/CustomMetadataLoader#60]
<img width="962" alt="Capture d’écran 2019-04-16 à 14 49 33" src="https://user-images.githubusercontent.com/6939326/56211058-65ed9100-6057-11e9-87bd-20c175de9c79.png">

Hello
I tried deploying CustomMetadataLoader into a production org, it didn't work and I am getting this error as a result. Can you provide some help with this issue?

**Comment by M-Cake** [comment]
yeah same issue here, doesn't want to use Heroku app to do so because I have some tests that will fail this way, and can't deploy it with WB either because of this error.

Did you find a solution ?

**Comment by karenbyrum** [comment]
I had the same issue. You have to unzip that zip file and create a new zip without the sample file. After I did that, it deployed without issue.
